uint16[3] border_color
float64[2] center_point
float64 width
float64 height
uint8 thickness
